src/components/LoadingOverlay/README.md

```diff
@@ -0,0 +1,32 @@
+<!--GITHUB_BLOCK-->
+
+# Loader
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {LoadingOverlay} from '@gravity-ui/uikit';
+```
+
+The `LoadingOverlay` component renders an overlay with a loader over the content. It can be used to preserve the desired layout while loading data.
+
+### PropTypes
+
+| Name             | Type               | Required | Default | Description                                 |
+| :--------------- | :----------------- | :------: | :------ | :------------------------------------------ |
+| className        | `String`           |          |         | CSS class name of root element              |
+| contentClassName | `String`           |          |         | CSS class name of a content wrapper element |
+| loading          | `Boolean`          |          | `false` | Overlay visibility state                    |
+| background       | `"base"` `"float"` |          | `base`  | Background style                            |
+| loaderProps      | `LoaderProps`      |          |         | Props of a Loader element                   |
+| children         | `React.ReactNode`  |          |         | Content                                     |
+| ...props         | `BoxProps`         |          |         | Props of a wrapping Box element             |
+
+### Basic usage
+
+```jsx
+<LoadingOverlay loading={loading} contentClassName={b('content')}>
+  <div>I am an example</div>
+  <div>content</div>
+</LoadingOverlay>
+```
```

Comment:
need to add some info about override background by `--g-color-base-background` css custom property

and examples with loader overrides

Reply:
Added examples for different loaders, as for overriding `--g-color-base-background`, as I remember, we've decided that it's not really appreciated by designers?